docs/modules/ROOT/pages/how-to/monitor-pvc-usage.adoc

```diff
@@ -0,0 +1,107 @@
+= Monitor PVC Disk Usage
+
+== Overview
+
+{product} exposes `kubelet` metrics that can be used to monitor PVC disk usage.
+
+The following volume metrics are available:
+
+[source]
+----
+kubelet_volume_stats_available_bytes
+kubelet_volume_stats_capacity_bytes
+kubelet_volume_stats_used_bytes
+
+kubelet_volume_stats_inodes
+kubelet_volume_stats_inodes_free
+kubelet_volume_stats_inodes_used
+----
+
+== Prerequisites
+
+* Command Line Access to APPUiO Cloud
+
+== Create a Disk Usage Alert
+
+The following instructions create two alerts that trigger when the disk usage or used https://en.wikipedia.org/wiki/Inode[inodes] of a PVC exceed 90%.
+
+. Set the namespace your application is running in
++
+[source,bash]
+----
+APP_NAMESPACE="my-app"
+----
+
+. Create the alert
++
+[source,shell]
+----
+kubectl -n "${APP_NAMESPACE}" apply -f - <<YAML
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: my-app-volume-alerts
+spec:
+  groups:
+  - name: my-app-volume.rules
+    rules:
+    - alert: VolumeAvailableSpaceLow
+      expr: |
+        (
+          sum(kubelet_volume_stats_available_bytes{persistentvolumeclaim!=""}) by (persistentvolumeclaim)
+        /
+          sum(kubelet_volume_stats_capacity_bytes{persistentvolumeclaim!=""}) by (persistentvolumeclaim)
```

Comment:
the `sum by (persistentvolumeclaim) ()` are technically not necessary, but probably don't hurt.

Reply:
I'm pretty sure it's needed if volume changes between nodes and there's some overlap due to the scrape interval.

Maybe it should be `max/min` though.

Also i need to test RWX volumes...

Reply:
> I'm pretty sure it's needed if volume changes between nodes and there's some overlap due to the scrape interval.

The query is not failing but can trigger N alerts for each mounted node.

Reply:
> > I'm pretty sure it's needed if volume changes between nodes and there's some overlap due to the scrape interval.
> 
> The query is not failing but can trigger N alerts for each mounted node.

right, but then using `sum()` would also report wrong absolute numbers for RWX volumes which are mounted on multiple nodes, so I think using `max()` would make more sense

Reply:
The supplied config is correct as `sum() / sum()` is fine. The tip is also correct but can trigger multiple alerts.

I'll add `max without(instance,node)` to the absolute value tip.